ares/n64/cpu/cpu.cpp

```diff
@@ -132,11 +135,7 @@ auto CPU::instructionPrologue(u64 address, u32 instruction) -> void {
 
 template<bool Recompiled>
 auto CPU::instructionEpilogue() -> void {
-  if constexpr(Recompiled) {
-    //simulates timings without performing actual icache loads
-    icache.step(ipu.pc, devirtualizeFast(ipu.pc));
-    assert(ipu.r[0].u64 == 0);
```

Comment:
A little concerned about losing this assert in debug builds. I guess we could replace it with a check in the JITed code. Not saying this has to be addressed in this PR, but I wanted to make note of it.